tyk-gateway.pid

```diff
@@ -1 +1 @@
-43267
\ No newline at end of file
+69425
```

Comment:
I think we should discard this

Reply:
I added this file into the gitignore file, it was introduced by mdcb testing framework and we dont need to keep track of changes on this file